tests/foreman/longrun/test_remote_DB.py

```diff
@@ -1,239 +0,0 @@
-
-    :Steps:
-
-        1. Run satellite-installer with pulp, foreman and candlepin db options set
-        2. Run satellite-installer --reset
-
-    :expectedresults:
-
-        1. Successful installation, all services running
-        2. Successful reset, all services running
-
-    :CaseLevel: System
-
-    :CaseComponent: Installer
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_offline_backup_external_all_with_SSL():
-    """Perform offline backup of external database
-
-    :id: dc452b72-66b8-4a69-adb1-da089a0d4942
-
-    :Steps: Run satellite-maintain backup offline
-
-    :expectedresults: Database backup created
-
-    :CaseLevel: System
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_online_backup_external_all_with_SSL():
-    """Perform online backup of external database
-
-    :id: 2781aeb7-742e-4c95-ad57-c2d1c025bf90
-
-    :Steps: Run satellite-maintain backup online
-
-    :expectedresults: Database backup created
-
-    :CaseLevel: System
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_snapshot_backup_external_all_with_SSL():
-    """Perform snapshot backup of external database
-
-    :id: e2d35b31-15ce-40da-bbbf-d302838d41b6
-
-    :Steps: Run satellite-maintain backup snapshot
-
-    :expectedresults: Database backup created
-
-    :CaseLevel: System
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_restore_offline_backup_external_all_with_SSL():
-    """Restore offline backup of external databases
-
-    :id: c3e38e31-0058-48e9-b0f2-59788df64c38
-
-    :Steps: Run satellite-maintain restore - and provide it offline backup
-
-    :expectedresults: Database backup restored
-
-    :CaseLevel: System
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_restore_online_backup_external_all_with_SSL():
-    """Restore online backup of external databases
-
-    :id: 5c724e98-af13-4d5f-ba2e-84c8767c0222
-
-    :Steps: Run satellite-maintain restore
-
-    :expectedresults: Database backup restored
-
-    :CaseLevel: System
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_restore_snapshot_backup_external_all_with_SSL():
-    """Restore snapshot backup of external databases
-
-    :id: e6439ddf-4f5a-450d-8911-73082589105d
-
-    :Steps: Run satellite-maintain restore
-
-    :expectedresults: Database backup restored
-
-    :CaseLevel: System
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_offload_internal_db_to_external_db_host():
-    """Offload internal databases content to an external database host
-
-    :id: d07235c8-4584-469a-a87d-ace4dadb0a1f
-
-    :Steps: Run satellite-installer with foreman, candlepin and pulp settings
-        pointing to external host
-
-    :expectedresults: Installed successful, all services running
-
-    :CaseLevel: Integration
-
-    :CaseComponent: Installer
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier1
-def test_upgrade_satellite_eith_all_external_db_SSL():
```

Comment:
Do we have this covered in upgrades? If not, we might want to move this test case to `test_offload_DB.py` as well.

Reply:
@vsedmik We have a Jenkins job running for remotedb upgrades, so yes, it's covered as well :)